disputer/disputer.js

```diff
@@ -57,8 +57,7 @@ class Disputer {
       } catch (error) {
         this.logger.error({
           at: "Disputer",
-          message:
-            "Cannot dispute liquidation: not enough collateral (or large enough approval) to initiate dispute.✋",
+          message: "Cannot dispute liquidation: not enough collateral (or large enough approval) to initiate dispute✋",
```

Comment:
Just curious, why didn't you have to change the logger or its transports to get the warn log levels to work the way we expect?

Reply:
Winston has 6 levels of logging in it by default. before we were just using 3 of 6. now we use 4 of 6. No extra customization needed :)